l2tdevtools/helpers/project.py

```diff
@@ -58,7 +58,8 @@ class ProjectHelper(cli.CLIHelper):
           'joachimmetz',
           'Onager',
           'rgayon']),
-      'PlasoScaffolder': frozenset([
+      'L2tScaffolder': frozenset([
+          'kiddinn',
           'joachimmetz',
```

Comment:
@kiddinn feel free to remove me from the L2tScaffolder default reviewers list

Reply:
done